drivers/base/bootvid/common.c

```diff
@@ -15,8 +15,59 @@ ULONG VidpScrollRegion[4] =
     SCREEN_HEIGHT - 1
 };
 
+PALETTE_ENTRY DefaultPalette[MAX_COLORS] =
+{
+    {0x00, 0x00, 0x00}, // Black
+    {0x00, 0x00, 0x80}, // Blue
+    {0x00, 0x80, 0x00}, // Green
+    {0x00, 0x80, 0x80}, // Cyan
+    {0x80, 0x00, 0x00}, // Red
```

Comment:
Default palette differs on different platforms.
<details>
  <summary>Showing the default palette (VirtualBox)</summary>

![Pal](https://user-images.githubusercontent.com/37072976/81494790-5037b300-92cd-11ea-945a-4e4f11d88996.png)
</details>

Color `4` is a BSoD color.
https://git.reactos.org/?p=reactos.git;a=blob;f=ntoskrnl/ke/bug.c;hb=592bafd60a33b82e62174f124ce80b2e0374a03a#l629
I suggest to move DefaultPalette back to the VGA code.

Reply:
Palette definitions in separate architecture files were almost identical, except the RGB color ranges were adapted for each specific protocol or chipset. I just made a common default palette for them all, with runtime conversion that is different for different platforms. Also I tested my changes, boot logo and BSoD are displayed correctly.

Reply:
I think blue background with color index 4 works because default palette gets modified elsewhere.

Reply:
Yes when displaying the boot logo the palette is set to the one specified in the bitmaps.

Reply:
Sorry, I misunderstood. I'm ok with it now.

Reply:
There was indeed problem with default palette, now it's fixed, thanks @disean